docs/release-notes/r6.md

```diff
@@ -0,0 +1,112 @@
+---
+id: r6
+title: Release 6
+sidebar_label: Release 6
+description: Release 6 features.
+keywords:
+  - docs
+  - polygon id
+  - holder
+  - issuer
+  - verifier
+  - wallet sdk
+  - release 6
+---
+
+## Environments
+
+https://www.notion.so/polygontechnology/Environments-4e9ad9f1426e4590813d955252b1397e
+
+## [Verifier API (1.0.0)](https://github.com/0xPolygonID/verifier-backend/releases/tag/v1.0.0)
+
+Reference verifier implementation, designed to be used as a starting point for people who want to build their own verifier.
+
+- Source code: https://github.com/0xPolygonID/verifier-backend
+- URL: https://verifier-backend.polygonid.me
+
+## [Marketplace (1.1.0)](https://github.com/0xPolygonID/marketplace/releases/tag/v1.1.0)
+
+1. Implement the reset password flow.
+
+## [Issuer Node (2.4.1)](https://github.com/0xPolygonID/issuer-node/releases/tag/v2.4.1)
+
+1. Pagination in the get credentials api endpoint
+2. Backward support for old raw QRCodes in API UI
+3. Support for credential refresh service
+4. Pagination in the getConnections endpoint
+5. Added revocation push notification
+6. Add qr code switcher
+7. UI can create credentials with refresh service field
+8. Support display method
+
+## [Schema Builder (2.0.0)](https://github.com/0xPolygonID/schema-builder/releases/tag/v2.0.0)
+
+1. Support for creating non merklized credentials
+2. Allow the selection of a custom IPFS gateway
+3. Added Proof Type and Circuit ID inputs
+4. Implemented v3 circuits and proof of uniqueness in the form
+5. Return API timestamps expressed in UTC
+6. Implement off-chain verifications
+7. Implement on-chain verifications
+
+## Mobile app (2.2.9) - [Android](https://play.google.com/store/apps/details?id=com.polygonid.wallet) - [iOS](https://apps.apple.com/us/app/polygon-id/id1629870183)
+ 
+1. Refresh credential service
+2. DisplayType
+3. Revoke credential from push notification
+4. Iden3commRevocationStatusV1.0 credential status type support
+5. Circuit V3 (experimental)
+6. Minor improvements
+
+## [Mobile SDK (2.2.9)](https://github.com/0xPolygonID/polygonid-flutter-sdk/releases/tag/v2.2.9)
+
+1. Refresh credential service
+2. DisplayType
+3. Credential status update iden3message
+4. Circuit V3 support
```

Comment:
```suggestion
4. Included setting to support V3 of circuits (which are currently in beta)
```

Reply:
Not really as this is the SDK, I'll add this exact phrase to the Mobile App.